lighthouse-core/gather/gather-runner.js

```diff
@@ -224,6 +282,13 @@ class GatherRunner {
 
     passes = this.instantiateGatherers(passes, options.config.configDir);
 
+    const gathererResults = {};
+    passes.forEach(pass => {
+      pass.gatherers.forEach(gatherer => {
+        gathererResults[gatherer.name] = [];
```

Comment:
yeah, I guess my thought was consistency, if for some reason we ever added a method before `beforePass` it would still work. Not sure why that seemed important. Updated.